product_examples/errors-and-omissions/products/errors-omissions/policy/exposures/professional_indemnity/perils/a_legal_liability.premium.liquid

```diff
@@ -0,0 +1,79 @@
+{% assign policy_v = data.policy_characteristics.field_values %}
+{% assign exp_v = data.exposure_characteristics.field_values %}
+{% assign peril_c = data.peril_characteristics %}
+
+{% assign premium = exp_v.revenuethisyear | times: 0.001 | round %}
+
+{% assign rate = "professional_indemnity_rate_table" | lookup: exp_v.professionalindemnity_type %}
+{% assign premium = premium | times: rate | round %}
+
+{% assign staterate = "state_rate_table" | lookup: exp_v.office_domicile_state %}
+{% assign premium = premium | times: staterate | round %}
+
+{% comment %} scale Legal Liability premium according to subcontract% {% endcomment %}
+{% assign subcontract = exp_v.subcontractors_percent %}
+
+
+{% if subcontract == "1-10%" %}
+   {% assign subcontract = 1.15 %}
+{% elsif subcontract == "10-20%" %}
+   {% assign subcontract = 1.5 %}
+{% elsif subcontract == "20-30%" %}
+   {% assign subcontract = 2.5 %}
+{% elsif subcontract == "30-40%" %}
+   {% assign subcontract = 3.5 %} 
+{% elsif subcontract == "40-50%" %}
+   {% assign subcontract = 4.5 %}
+{% elsif subcontract == "More than 50%" %}
+   {% assign subcontract = 6 %}    
+{% else %}
+   {% assign subcontract = 1 %}
+{% endif %}
+
```

Comment:
I'm not sure where to put this exactly because its not a specific line of code, but when when creating a test policy for a company with annual revenues of 10,000, it gives a premium of almost 1000. Is this a generally reasonable premium (10% of total revenues) for a company to pay for E&O? (This issue only exists for lower revenue companies. I'm not sure if this is totally expected or if the math needs to be changed, cut just wanted to point it out).